docs/pages/workflow/overview.mdx

```diff
@@ -139,6 +139,8 @@ Note that you can use [config plugins](/config-plugins/introduction/) to modify
 
 If you add a new native dependency to your project or change the project configuration in Expo app config (**app.json/app.config.js**), you can run `npx expo prebuild --clean` to re-generate the native project directories.
 
+To determine if a new dependency changes native project directories, you can check that library's installation instructions. For example, for Android, it is common to apply changes from a library's instructions to **android/app/src/main/AndroidManifest.xml**. Similarly, for iOS, run `pod install` to update the **ios/Podfile** or update the **Info.plist** directly to change the project configuration.
```

Comment:
i'd prefer for us to improve the explanation on the "[Using libraries](https://docs.expo.dev/workflow/using-libraries/)" page (which is currently maybe a bit too focused towards Expo Go) and then linking out that from here

Reply:
(i don't want to block this pr on this comment though, we can follow up on it later)

Reply:
@brentvatne, I've updated the section in Using libraries and included the context about development builds and the factors to determine whether a library includes changes to native code config or not.